addons/kommander/1.2/kommander.yaml

```diff
@@ -18,8 +18,11 @@ metadata:
     docs.kubeaddons.mesosphere.io/karma: "https://github.com/prymitive/karma"
     docs.kubeaddons.mesosphere.io/kommander-grafana: "https://grafana.com/docs/"
     values.chart.helm.kubeaddons.mesosphere.io/kommander: "https://raw.githubusercontent.com/mesosphere/charts/d71f1e0/stable/kommander/values.yaml"
-    helmv2.kubeaddons.mesosphere.io/upgrade-strategy: '[{"upgradeFrom": "<=0.1.22",
```

Comment:
this line break was added by the automated chart bump job, if we would rather it not be adding line breaks maybe we can ask @d2iq-dispatch to fix the script